kitsune/questions/views.py

```diff
@@ -462,25 +462,6 @@ def aaq(request, product_key=None, category_key=None, step=1):
 
     template = "questions/new_question.html"
 
-    # Check if any product forum has a locale in the user's current locale
```

Comment:
This change is working fine when the user starts the AAQ flow from `/questions/new`. However if you click the `Get Help` link and there isn't an AAQ flow then the KB is shown directly. Is this the intention here?